index.html

```diff
@@ -24,9 +24,11 @@
             <ul class="menutabs">
                 <a href="#"><li>Portfolio</li></a>
                 <a href="#"><li>About</li></a>
-                <a href="#"><li>Portfolio</li></a>
+                <a href="#"><li>Contact</li></a>
```

Comment:
- Hey, please, kindly double-check to add the missing opening `<li>` tags as it's currently missing to improve your document structure.

Reply:
Not sure if this applies, all the tags have their opening and closing tag